pages/common/mvn.md

```diff
@@ -0,0 +1,25 @@
+# mvn
+
+> Runnign Apache Maven.
+
+- Most common usage is to invoke a life cicle phase:
```

Comment:
We do not use multiple commands for a single example. If this is the most common use case, I would recommend to split this into 3 separate examples. And probably you can remove the other examples which are not so commonly used.

Reply:
I have remove the two extra lines, it worths less that the next examples.